### PR TITLE
MAUI-4918-Create-UG-for-Error-Bar-and-waterfall-Series-in-MAUI

### PR DESCRIPTION
**Pull request by SuryaKaran2143**
### Description ###
 
User Guide for the Error Bar Series and Waterfall Series in MAUI Platform

### What problem it solves? ###
All about the Error bar Series feature and its types. And Initialising the error Bar Series was explained here.
All about the waterfall Series feature And Initialising the waterfall Series was explained here.
 
 
### Solution description ###
 
- I Created the UG Document in md Format named as ErrorBar.md
- I attached a folder contains the Chart_types Images for Error Bar Series
- I attached the images to the md file
- I Created the UG Document in md Format named as Waterfall.md
- I attached a folder contains the Chart_types Images for Waterfall Series
- I attached the images to the md file
- I added the Waterfall Series in the html List file

### Output Screenshot ###
 
BasicRenderingErrorBar
BasicRendering
NegativePointsBrush
SummaryBindingPath

### CODE REVIEW

File: MAUI/Cartesian-Charts/Errorbar-Chart.md [hunk 1]
```diff
@@ -0,0 +1,762 @@
...
+
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Error Bar Series in MAUI Chart](Chart-types_images/BasicRenderingErrorBar.png)
+
+## Mode 
+This [Mode]() property defines whether to identify a horizontal error or vertical error. By default, the Mode value is [Both](), which will display both horizontal and vertical error values.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Errorbar-Chart.md:82]: The error bar mode specifies whether the error bar should be drawn horizontally, vertically or both. The [Mode]() property used to switch the error bar mode.

File: MAUI/Cartesian-Charts/Errorbar-Chart.md [hunk 2]
```diff
@@ -0,0 +1,762 @@
...
+
+
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Vertical Mode Support in Error Bar Series](Chart-types_images/ModeVertical.png)
+
+
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Errorbar-Chart.md:164]: remove unwated line space.

File: MAUI/Cartesian-Charts/Errorbar-Chart.md [hunk 3]
```diff
@@ -0,0 +1,762 @@
...
+## Direction 
+
+[ErrorBar]() series allows you to view the horizontal and vertical error values in both positive and negative directions.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Errorbar-Chart.md:167]: The HorizontalDirection() and VerticalDirection() specifies whether to show positive, negative, or both directions of error values.

Comment (Saravanan-Madhesh) [reply MAUI/Cartesian-Charts/Errorbar-Chart.md:167]: ErrorBarDirection() contains below values:
*`Both` -  It indicates the actual data point value along with specific amount of positive and negative error values.

*`Plus` -  It indicates the actual data point value along with specific amount of positive error value.

*`Minus`-  It indicates the actual data point value along with specific amount of negative error value.

File: MAUI/Cartesian-Charts/Errorbar-Chart.md [hunk 4]
```diff
@@ -0,0 +1,762 @@
...
+
+*`Fixed`
+
+*`Percentage`
+
+*`Standard Error`
+
+*`Standard Percentage`
+
+N> The default error bar series Type is Fixed.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Errorbar-Chart.md:439]: Mention error values should be set if the type is not custom

File: MAUI/Cartesian-Charts/Waterfall-Chart.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+title: Waterfall Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about the Waterfall Series chart and its features in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Waterfall Series in .NET MAUI Chart
+
+## Waterfall Series
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Waterfall-Chart.md:12]: Waterfall Chart

File: MAUI/Cartesian-Charts/Waterfall-Chart.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+    series.YBindingPath = "Value";
+    chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Waterfall Chart in MAUI Chart](Chart-types_images/BasicRendering.png)
+
+### SummarySegmentPath and SummaryPointsBrush
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Waterfall-Chart.md:62]: REmove the changes as we discussed.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Errorbar-Chart.md [hunk 7]
```diff
@@ -0,0 +1,762 @@
+---
+layout: post
+title: Error Bar Series Chart in .NET MAUI Chart Control | Syncfusion
```
[the remark this answers is not in the document]
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Errorbar-Chart.md:3]: I change the Sentence

File: MAUI/Cartesian-Charts/Errorbar-Chart.md [hunk 8]
```diff
@@ -0,0 +1,762 @@
...
+description: Learn here all about error bar chart support in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Error Bar Series in .NET MAUI Chart
```
[the remark this answers is not in the document]
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Errorbar-Chart.md:10]: I modified the title

File: MAUI/Cartesian-Charts/Errorbar-Chart.md [hunk 9]
```diff
@@ -0,0 +1,762 @@
...
+
+[ErrorBarSeries]() indicates the errors or uncertainty in reported values. This will find the possible variations in measurements, and in Chart control these values are displayed as data points.
+The [HorizontalErrorValue]() and the [VerticalErrorValue]() is used to set the error value(variation) to the series.
+
+N> The Error Bar Series Chart is not an individual Chart, it associate with a main Chart. Here, we use Scatter Series Chart as Main Chart with the Error Bar Series Chart Support
```
[the remark this answers is not in the document]
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Errorbar-Chart.md:15]: I resolved this

File: MAUI/Cartesian-Charts/Errorbar-Chart.md (hunk 1 above)
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Errorbar-Chart.md:82]: I changed the mode Property

File: MAUI/Cartesian-Charts/Errorbar-Chart.md (hunk 3 above)
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Errorbar-Chart.md:167]: I replaced the direction part

File: MAUI/Cartesian-Charts/Errorbar-Chart.md (hunk 2 above)
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Errorbar-Chart.md]: I removed the unwanted spaces

File: MAUI/Cartesian-Charts/Waterfall-Chart.md (hunk 5 above)
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Waterfall-Chart.md]: I chaged the headings

File: MAUI/Cartesian-Charts/Waterfall-Chart.md (hunk 6 above)
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Waterfall-Chart.md]: i resolved this

File: MAUI/Cartesian-Charts/Errorbar-Chart.md [hunk 10]
```diff
@@ -0,0 +1,579 @@
...
+    };
+
+    chart.Series.Add(series);
+    chart.Series.Add(errorBar);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Error Bar Series in MAUI Chart](Chart-types_images/BasicRenderingErrorBar.png)
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Errorbar-Chart.md:84]: image name change like 

MAUI_ErrorBar_featurname

Change this for all images.

File: MAUI/Cartesian-Charts/Errorbar-Chart.md [hunk 11]
```diff
@@ -0,0 +1,579 @@
...
+        VerticalDirectyion="Minus"
+    };
+
+    chart.Series.Add(errorBar);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Horizontal Direction as Both in Error Bar Series](Chart-types_images/BasicRenderingErrorBar.png)
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Errorbar-Chart.md:252]: The code snippet and image were different, add a image which describe in the code snippet.

File: MAUI/Cartesian-Charts/Errorbar-Chart.md [hunk 12]
```diff
@@ -0,0 +1,579 @@
...
+
+*`Fixed`
+
+*`Percentage`
+
+*`Standard Error`
+
+*`Standard Percentage`
+
+*`Custom`
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Errorbar-Chart.md:266]: Instead of bullet point update in sentence.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Errorbar-Chart.md (hunk 4 above)
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Errorbar-Chart.md]: I resolved this issue

File: MAUI/Cartesian-Charts/Errorbar-Chart.md (hunk 12 above)
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Errorbar-Chart.md]: I chaged it

File: MAUI/Cartesian-Charts/Errorbar-Chart.md (hunk 11 above)
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Errorbar-Chart.md]: I replace the images

File: MAUI/Cartesian-Charts/Errorbar-Chart.md (hunk 10 above)
Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Errorbar-Chart.md]: I replace image names

File: MAUI/Cartesian-Charts/Waterfall-Chart.md [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+documentation: ug
+---
+
+# Waterfall Chart in .NET MAUI Chart
+
+## Waterfall Chart
+
+[WaterfallSeries]() clarifies the cumulative effect of a set of provided positive and negative values. The series is represented by a rectangle and a connector between the rectangles. To render a waterfall chart, create an instance of [WaterfallSeries](), and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of the [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1)
+
+*`SummaryBindingPath`- gets or sets the string value that indicates the sum of previous segments in the series.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Waterfall-Chart.md:16]: image

@SuryaKaran2143 while explaining properties use above format. 

example
[SummaryBindingPath]() of the string type is used to get or set the path value on the source subject to serve summary values to the series.

Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Waterfall-Chart.md]: I Solved this issuue

**Review by Saravanan-Madhesh — APPROVED** [review]

**Pull request merged**